### PR TITLE
Docs: Expand ruler recommended alerting docs

### PR DESCRIPTION
**Pull request by ianbillett**
* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

This addition to the documentation would have answered [this](https://cloud-native.slack.com/archives/CK5RSSC10/p1572959144493000) question that I raised on slack.

Added sentence to the documentation for `Ruler` component.

## Verification

Not applicable.

### CODE REVIEW

File: docs/components/rule.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The most important metrics to alert on are:
 indicate connection, incompatibility or misconfiguration problems.
 
 * `prometheus_rule_evaluation_failures_total`. If greater than 0, it means that that rule failed to be evaluated, which results in
-either gap in rule or potentially ignored alert. Alert heavily on this if this happens for longer than your alert thresholds.
+either gap in rule or potentially ignored alert. This metrics indicates an unhealthy downstream query component. Alert heavily on this if this happens for longer than your alert thresholds.
```
Comment (bwplotka) [inline docs/components/rule.md]:
```suggestion
either gap in rule or potentially ignored alert. This metric might indicate problems on the queryAPI endpoint you use.

Alert heavily on this if this happens for longer than your alert thresholds.
```

Comment (ianbillett) [reply docs/components/rule.md]: SGTM!

**Comment by ianbillett** [comment]
@FUSAKLA thanks for the review 😄- I've accepted @bwplotka's suggestion.

File: docs/components/rule.md [hunk 2]
```diff
@@ -81,7 +81,9 @@ The most important metrics to alert on are:
 indicate connection, incompatibility or misconfiguration problems.
 
 * `prometheus_rule_evaluation_failures_total`. If greater than 0, it means that that rule failed to be evaluated, which results in
-either gap in rule or potentially ignored alert. Alert heavily on this if this happens for longer than your alert thresholds.
+either gap in rule or potentially ignored alert. This metric might indicate problems on the queryAPI endpoint you use.
+
```
Comment (squat) [inline docs/components/rule.md]: This new line here breaks the formatting of the list. This makes the line “alert heavily...” appear out of the list, and then the list continues.

**Comment by ianbillett** [comment]
@squat good spot - updated to remove the newline

**Comment by squat** [comment]
Thanks :)

**Pull request merged**